flink-connectors/flink-connector-kinesis/src/main/resources/META-INF/NOTICE

```diff
@@ -55,7 +55,7 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.typesafe.netty:netty-reactive-streams-http:2.0.5
 - com.typesafe.netty:netty-reactive-streams:2.0.5
 - commons-logging:commons-logging:1.1.3
-- com.fasterxml.jackson.core:jackson-core:2.13.0
+- com.fasterxml.jackson.core:jackson-core:2.13.2
```

Comment:
Could we actually bump this to 2.13.2.2 ? Looks like 2.13.2 still has a vulnerability according to https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind